source/user/ci.rst

```diff
@@ -8,7 +8,7 @@ PySCF has separate molecular implementations for configuration
 interaction singles and doubles (CISD) (:mod:`ci`) and full
 configuration interaction (FCI) (:mod:`fci`). The functionalities of the
 CISD implementation are similar to the functionalities of MP2
-(:numref:`user_mp2`) and CCSD (:numref:`user_cc`) in PySCF.
```

Comment:
This should stay numref, maybe a merge change in the wrong direction :D.

Reply:
This was intentional to remove the section numbers in the side-bar table of contents